src/sql/migrations/index.ts

```diff
@@ -0,0 +1,54 @@
+import { exit } from 'process'
+import { pg } from '../../connections/postgres'
+import { postgesLog as log } from '../../connections/loggers'
+import { isEmptyArray } from '@subsocial/utils'
+import { readFileSync, readdirSync } from 'fs'
+
+const RELATION_NOT_FOUND_ERROR = '42P01'
+const updateSchemaVersionQuery = 'UPDATE df.schema_version SET value = $1 WHERE value = $2'
+
+export const INIT_FILE = readFileSync(`${__dirname}/1-init.sql`, 'utf8')
+
+export enum MigrationStatus {
+  NewMigrationsExecuted = "New migrations were successfuly executed",
+  NoNewMigrations = "No new migrations for the current database schema",
+  SchemaRestored = "Schema was restored from 1-init",
+}
+
+const stripSchemaVersion = (fileName: string) => parseInt(fileName.split('-')[0])
+
+export const getMigrationStatus = async (): Promise<MigrationStatus> => {
```

Comment:
Bad name of the function.
Consider: `runDbMigrations()` or `migrateDb()` or `upgradeDbSchema()`